tests/test_bitaxe_monitor.py

```
@@ -1,144 +1,362 @@
+Comprehensive tests for Enhanced BitAxe Monitor
 
-This module contains basic tests for the Bitaxe monitoring system.
+This module contains tests for both the classic and enhanced monitoring systems.
 """
 
 import unittest
 import sys
 import os
+import json
+import tempfile
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)